dspace/config/local.cfg.EXAMPLE

```diff
@@ -85,18 +85,41 @@ db.password = dspace
 #      so this may be set to ${db.username} in most scenarios.
 db.schema = public
 
-## Connection pool parameters
+## Database Connection pool parameters
+# DSpace wraps around the Apache Commons DBCP2 component, to read about its configuration
+# see: https://commons.apache.org/proper/commons-dbcp/configuration.html
 
-# Maximum number of DB connections in pool (default = 30)
-#db.maxconnections = 30
+# Maximum number of active DB connections in pool (-1 = unlimited)
+# (default = 30)
+db.maxconnections = 30
 
-# Maximum time to wait before giving up if all connections in pool are busy (milliseconds)
+# Maximum time to wait before giving up if all connections in pool are busy (milliseconds), (-1 = unlimited)
 # (default = 5000ms or 5 seconds)
-#db.maxwait = 5000
+db.maxwait = 5000
+
+# Minimum number of idle connections in pool
+# (default = 0)
+db.minidle = 0
```

Comment:
Starting here, I'm not sure whether we need to include *ALL* of these settings in the `local.cfg.EXAMPLE`.  In order to keep `local.cfg.EXAMPLE` small, we tend to only include settings that **most people** will want/need to update.  So, while I'm not against copying all of these into here...it seems like these are *advanced* settings which should be documented, but not touched by most people?